AEPMessaging/Sources/MessagingConstants.swift

```diff
@@ -216,6 +216,7 @@ enum MessagingConstants {
                 static let DISPLAY = "decisioning.propositionDisplay"
                 static let INTERACT = "decisioning.propositionInteract"
                 static let DISMISS = "decisioning.propositionDismiss"
+                static let DISQUALIFY = "decisioning.propositionDisqualify"
```

Comment:
Is `decisioning.propositionDisqualify` an actual recognized Edge EventType that we will use in the future?
or is this added to make MessagingEdgeEventType enum implementation complete ?

Reply:
not sure yet - we aren't actually sending it today but i did add it for completeness of the enum